src/test/java/kitchenpos/TableAcceptanceTest.java

```diff
@@ -0,0 +1,198 @@
+    }
+
+    @DisplayName("잘못된 ")
+    @Nested
+    class FailTest {
+
+        @DisplayName("주문 테이블 id로 테이블을 비우거나, 채우려고 하면 500 응답을 받는다.")
+        @ValueSource(longs = {-1L, 2000L})
+        @ParameterizedTest
+        void fail1(final long tableId) {
+
+            OrderTable orderTable = new OrderTable();
+            orderTable.setEmpty(true);
+
+            //when
+            ExtractableResponse<Response> response = 주문_테이블_비우거나_채우기(orderTable, tableId);
+
+            //then
+            assertThat(response.statusCode()).isEqualTo(HttpStatus.INTERNAL_SERVER_ERROR.value());
+        }
+
+        @DisplayName("손님의 수로 변경하려고 하면, 500 응답을 받는다.")
+        @Test
+        void fail2() {
+            //given
+            OrderTable orderTable = new OrderTable();
+            orderTable.setEmpty(false);
+            주문_테이블_비우거나_채우기(orderTable, 1L);
+
+            orderTable = new OrderTable();
+            orderTable.setNumberOfGuests(-1);
+
+            //when
+            ExtractableResponse<Response> response = 손님_수_변경(orderTable, 1L);
+
+            //then
+            assertThat(response.statusCode()).isEqualTo(HttpStatus.INTERNAL_SERVER_ERROR.value());
+        }
+
+        @DisplayName("테이블 Id로 손님의 수를 변경하려고 하면, 500 응답을 받는다.")
+        @Test
+        void fail3() {
+            //given
+            OrderTable orderTable = new OrderTable();
+            orderTable.setEmpty(false);
+            주문_테이블_비우거나_채우기(orderTable, 1L);
+
+            orderTable = new OrderTable();
+            orderTable.setNumberOfGuests(1);
+
+            //when
+            ExtractableResponse<Response> response = 손님_수_변경(orderTable, 1000L);
+
+            //then
+            assertThat(response.statusCode()).isEqualTo(HttpStatus.INTERNAL_SERVER_ERROR.value());
+        }
+    }
+
+    @DisplayName("비어있는 테이블의 손님의 수를 변경하려고 하면, 500 응답을 받는다.")
+    @Test
+    void fail4() {
+        //given
+        OrderTable orderTable = new OrderTable();
+        orderTable.setNumberOfGuests(1);
+
+        //when
+        ExtractableResponse<Response> response = 손님_수_변경(orderTable, 1L);
+
+        //then
+        assertThat(response.statusCode()).isEqualTo(HttpStatus.INTERNAL_SERVER_ERROR.value());
+    }
+
+    public static ExtractableResponse<Response> 주문_테이블_비우거나_채우기(final OrderTable orderTable, final Long tableId) {
+        return RestAssured
+            .given().log().all()
+            .body(orderTable)
+            .contentType(MediaType.APPLICATION_JSON_VALUE)
+            .put(String.format("/api/tables/%s/empty", tableId))
+            .then().log().all()
+            .extract();
+    }
+
+    private ExtractableResponse<Response> 손님_수_변경(final OrderTable orderTable, final Long tableId) {
+        return RestAssured
+            .given().log().all()
+            .body(orderTable)
+            .contentType(MediaType.APPLICATION_JSON_VALUE)
+            .put(String.format("/api/tables/%s/number-of-guests", tableId))
```

Comment:
```suggestion
            .put("/api/tables/{tableId}/number-of-guests", tableId)
```
String의 format 함수를 따로 쓸 필요 없이, 위와 같이 변경할 수도 있을 것 같아요 😃

Reply:
아, 이게 모종의 이유로 안됐어가지고, String.format을 썻던것으로 기억하는데, 다시 바꿔보니 이상이 없네요 ㅎㅎ
제가 뭔가 착각하고 있었나봅니다!